Documentation/cmdref/cilium-agent.md

```diff
@@ -71,6 +71,7 @@ cilium-agent [flags]
       --enable-bpf-clock-probe                               Enable BPF clock source probing for more efficient tick retrieval
```

Comment:
Small typo in this commit message (2nd commit): `in order to select weather tail calls`.

Reply:
It's a thing here, the weather jumps from sunny to rainy in an instant. Stack is preserved and all. Famous weather tail calls.